dataminer-overview/Partner_Program/About_Partner_Program.md

```diff
@@ -0,0 +1,92 @@
+---
+uid: About_Partner_Program
+---
+
+# About the Partner Program
+
+At Skyline we value the collaboration with our partners and your commitment and contributions to create value to end customers. Our Partner Program is designed to recognize and reward you for the value your technical and business skills add to the market.
+
+The moment you become our partner you start receiving benefits.  
+
+- Buy DataMiner Licenses, Usage-based Services, and Support Services at reduced rates
+
+- Access lower certification and training fees
+
+- Referrals
+
+## Partner skill levels
+
+Your engagement is rewarded with DataMiner Credits, recognizing your expertise and collaboration. The number of credits obtained in a Fiscal Year determines your skill level, which remains valid from the moment you earn it until the end of the following year.
+
+There are 3 types of skill level depending on your engagement:
+
+- Member (bronze level)
+
+- Collaborator (silver level)
+
+- Creator (gold level)
+
+A partner can obtain these levels based on business skills, technical skills, or both.
+
+## Rewards
+
+Your skill level allows you to buy DataMiner Licenses, DataMiner Credits, and Support Services at discounted rates. The discount rates vary with your skill Level.
+
+*For example: As a partner, you achieved the Technical Creator level, while also having the Business Collaborator level. To determine your Base Discount, we look to the highest level, which in this example is the Technical Creator level.*
+
+The credits you accumulate serve a dual purpose. Not only do they allow you to achieve higher skill levels, you can also translate them to commercial value:
+
+- Sales transactions: Use credits to reduce software license or usage-based services (DataMiner Credits) prices when selling to customers, in addition to your level's Base Discounts.
+
+- Self-use: reduce your subscriptions costs, training, and certifications.
+
+> [!NOTE]
+> The total number of credits utilized in any given order shall not exceed 5% of the DataMiner Licenses or Usage-based Services ordered for an end customer, 25% of DataMiner Credits when ordering for self-use (e.g. setting a lab or a demo system), or up to 50%, for your training and certifications.
```

Comment:
Not sure if "shall" is OK here, since this is usually only used that way in strictly legal documents (e.g. contracts), and even then it's meant to express obligation on behalf of the subject (see [info here](https://englishforlawyers.ca/f/contracts-expressing-obligations-using-shall-will-and-must)), which doesn't make much sense here as the subject is the "total number of credits". I'd suggest this instead:
```suggestion
> The total number of credits utilized in any given order cannot exceed 5% of the DataMiner Licenses or Usage-based Services ordered for an end customer, 25% of DataMiner Credits when ordering for self-use (e.g. setting up a lab or a demo system), or 50% for your training and certifications.
```